lib/matplotlib/transforms.py

```diff
@@ -1778,11 +1778,7 @@ class Affine2DBase(AffineBase):
     """
     The base class of all 2D affine transformations.
 
-    2D affine transformations are performed using a 3x3 numpy array::
-
-        a c e
-        b d f
-        0 0 1
+    2D affine transformations are performed using a 3x3 numpy
```

Comment:
The info that the last line is 0, 0, 1 seems worth leaving in.

Reply:
I am going to look through those changes later. I just wanted a working example when I found out what cases the warnings.